library/src/main/java/andrewoid/neutralnetwork/NeuralNetworkComponent.java

```diff
@@ -27,11 +26,11 @@ public void paintComponent(Graphics g) {
         for (int i = 0; i < layers.length; i++) {
             for (int j = 0; j < layers[i].length; j++) {
                 neuronValue = (int) layers[i][j].getValue();
-                neuronColor = colors[neuronValue];
+                neuronColor = new Color(neuronValue*10);
                 drawNeuron(i, j, neuronColor, g);
                 for (int k = 0; k < layers[i][j].getNumWeights(); k++) {
                     pathWeight = (int) layers[i][j].getWeight(k);
-                    pathColor = colors[pathWeight];
+                    pathColor = new Color(pathWeight*100000);
```

Comment:
apparently, `pathWeight` could be positive or negative, and could be larger than 1.0.